### PR TITLE
py-eric-ide: new port (eric6), obsolete eric4

### PR DESCRIPTION
**Pull request by reneeotten**
#### Description
This PR adds ```py-eric-ide``` that provides ```eric6```, it also obsoletes the old ```eric4``` port. I've tried to integrate the attempts in the relevant Trac tickets ([52865](https://trac.macports.org/ticket/52865) and [56636](https://trac.macports.org/ticket/56636)) and PR #3638 to obtain a working port. 

It supports py27/py37 and Qt4/Qt5; ports for different Python versions can be installed side-by-side, of course one needs to pick a backend (default Qt5). For now I left @giansalvo as maintainer as he submitted the PR, but if @iEFdev or josephsacco (from the Trac tickets) want to be listed as well that would be fine. 

Please give this PR a try and let me know if there is anything I have missed; let's try to get this finalized!

###### Tested on

macOS 10.14.4 18E226
Xcode 10.2.1 10E1001
Python 2.7 + Qt4 & Python 3.7 + Qt5

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change? 
Closes/supersedes #3638 that seems stalled and has been unfinished in the PR queue for months.
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`? N/A
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Please can you explicit link to the trac tickets you refer to in the description above. I don’t want to have to go search trac myself to find them. Also, if this pr address some of those tickets, please use the ‘closes’ syntax to refer to them, so they automatically get closed when this gets merged.

**Comment by reneeotten** [comment]
> Please can you explicitly link to the trac tickets you refer to in the description above. I don’t want to have to go search trac myself to find them. Also, if this pr addresses some of those tickets, please use the ‘closes’ syntax to refer to them, so they automatically get closed when this gets merged.

well, the tickets were listed in the commit message already so not too hard to find. Nevertheless, I have added them now explicitly to the description.

**Comment by cjones051073** [comment]
Ah yes, sorry. I missed them in the extended commit message.

File: python/py-eric-ide/Portfile [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+name                py-eric-ide
+set real_name       eric6
+version             19.04
+revision            0
+
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             GPL-3+
+maintainers         {gmail.com:giansalvo.gusinu @giansalvo} openmaintainer
```
Comment (mf2k) [inline python/py-eric-ide/Portfile]: Has @giansalvo agreed to maintain this port? If not, please use `nomaintainer` unless you are willing to maintain this.

**Comment by mf2k** [comment]
This looks really good to me.

File: python/py-eric-ide/Portfile (hunk 1 above)
Comment (reneeotten) [reply python/py-eric-ide/Portfile]: I don't know... waiting for his response. If he doesn't answer in the next day or so, I'll remove him and use ```nomaintainer```.

Could you please try and install ```py27-eric-ide``` and see if it starts correctly for you? In [Trac ticket 56636](https://trac.macports.org/ticket/56636) Joseph says that for him it doesn't start, but I cannot reproduce the problem and suspect it is something in his setup...

Comment (giansalvo) [reply python/py-eric-ide/Portfile]: @reneeotten  please remove me as maintainer of this port

**Pull request merged**